featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -238,13 +239,17 @@ def __init__(self,
             primitive_options = {}
         all_primitives = self.trans_primitives + self.agg_primitives + \
             self.where_primitives + self.groupby_trans_primitives
+        if any(isinstance(entity.df, pd.DataFrame) for entity in self.es.entities):
+            if not all([Library.PANDAS in primitive.compatibility for primitive in all_primitives]):
+                bad_primitives = ", ".join([prim.name for prim in all_primitives if Library.PANDAS not in prim.compatibility])
+                raise ValueError('Selected primitives are incompatible with pandas EntitySets: {}'.format(bad_primitives))
```

Comment:
Small thing, but if you wanted, you could do
```suggestion
bad_primitives = ", ".join([prim.name for prim in all_primitives if Library.PANDAS not in prim.compatibility])
if bad_primitives:
    raise ValueError('Selected primitives are incompatible with pandas EntitySets: {}'.format(bad_primitives))
```

Reply:
The pattern could follow for the dask and koalas ones below

Reply:
Or something like
```
bad_primitives = [prim.name for prim in all_primitives if library not in prim.compatibility]
if bad_primitives:
    msg = "Selected primitives are incompatible with {} EntitySets: {}"
    raise ValueError(msg.format(library, ", ".join(bad_primitives))